### ISSUE TITLE
Sorry but another NullPointer here....

### ISSUE THREAD

**Issue by Jebula999** [Armarr/Autorank-2#234]
Here is a pasty to make it easier to read.

I've had the problem before but it has returned....

http://pastebin.com/GpjenaMu

**Comment by Staartvin** [comment]
Could you show me your whole startup log?

**Comment by Jebula999** [comment]
here is the startup, its rather long.

As well at the bottom is a few error reports from AR and Stats, but the errors only occurred after updating AR.

Thanx in advance for looking into this.

http://pastebin.com/JpDC3cbn

**Comment by Staartvin** [comment]
Can you give your config file?

**Comment by Jebula999** [comment]
http://pastebin.com/FmbMQ1g6

**Comment by Jebula999** [comment]
Here is the newest log.

There are more errors all over the place.

http://pastebin.com/WuycVs7D

**Comment by Staartvin** [comment]
Your latest pastebin was removed. I see problems with the 'mobs killed' requirement. This is fixed in the latest dev build. The latest dev build runs on Stats 3 however. Stats 3 is not very stable at the moment, but you could try it (make a backup of your server beforehand though).

You can get the dev builds from here: http://62.131.38.74:8080/job/Autorank/

**Comment by Jebula999** [comment]
Hey

When you say a problem with "mobs killed" do you know what the problem is?

I will try the fix you suggested and let you know.

Thanx hey

**Comment by Jebula999** [comment]
getting this now

[18:20:37] [Thread-19/INFO]: [Autorank] Could not find name of account with
uuid: '6e9dee95-42d7-3c2a-90e0-b9cd7c717da6'
[18:20:37] [Thread-19/WARN]: Unexpected token END OF FILE at position 0.
[18:20:37] [Thread-19/WARN]: at
org.json.simple.parser.JSONParser.parse(JSONParser.java:257)
[18:20:37] [Thread-19/WARN]: at
org.json.simple.parser.JSONParser.parse(JSONParser.java:81)
[18:20:37] [Thread-19/WARN]: at
org.json.simple.parser.JSONParser.parse(JSONParser.java:75)
[18:20:37] [Thread-19/WARN]: at
me.armar.plugins.autorank.util.uuid.NameFetcher.call(NameFetcher.java:98)
[18:20:37] [Thread-19/WARN]: at
me.armar.plugins.autorank.util.uuid.UUIDManager$1.run(UUIDManager.java:176)
[18:20:37] [Thread-19/WARN]: at java.lang.Thread.run(Thread.java:745)
[18:20:38] [Thread-20/INFO]: [Autorank] Could not find name of account with
uuid: '4ccc0fc7-4413-320a-9bce-bfedeb5c0e48'
[18:20:38] [Thread-20/WARN]: Unexpected token END OF FILE at position 0.
[18:20:38] [Thread-20/WARN]: at
org.json.simple.parser.JSONParser.parse(JSONParser.java:257)
[18:20:38] [Thread-20/WARN]: at
org.json.simple.parser.JSONParser.parse(JSONParser.java:81)
[18:20:38] [Thread-20/WARN]: at
org.json.simple.parser.JSONParser.parse(JSONParser.java:75)
[18:20:38] [Thread-20/WARN]: at
me.armar.plugins.autorank.util.uuid.NameFetcher.call(NameFetcher.java:98)
[18:20:38] [Thread-20/WARN]: at
me.armar.plugins.autorank.util.uuid.UUIDManager$1.run(UUIDManager.java:176)
[18:20:38] [Thread-20/WARN]: at java.lang.Thread.run(Thread.java:745)

**Comment by Jebula999** [comment]
after few restarts, the only error i seem to get now is this:

[19:26:06 WARN]: org.apache.commons.lang.UnhandledException: Plugin
Autorank v3.5.1-Dev-392 generated an exception while executing task 71
        at
org.bukkit.craftbukkit.v1_8_R1.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:56)
        at
java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
        at
java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
        at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.NullPointerException

**Comment by Staartvin** [comment]
Could you give me the whole NullPointerException error you're getting. What version of Autorank are you currently running?

**Comment by Jebula999** [comment]
I have updated the Autorank to the version you recommended i get, as well as updated STATS to v3.0 like you said it needs.

It seems to have broken other things when updating.

I'm going to link you my whole log file, you will see Autorank errors, STATS errors and AreaProtect errors since i updates AR and STATS.


If this carries on i will have to downgrade back to previous versions and hope for the best as it is more broken than what it was before updating....




Log file: https://www.dropbox.com/s/ohduw55iqnihrf8/latest.log?dl=0

The log file is long, but it has EVERYTHING in it so i don't have to keep using pastebin and things going wrong.

**Comment by Staartvin** [comment]
It seems to be a certain error that keeps repeating, so if we can fix that one error, you'll be good. I'm doing some checks and I hope to find the cause of this problem soon.

**Comment by Staartvin** [comment]
I have a new build for you, but my dev build server is acting a little weird. It'll be ready for download soon though.

**Comment by Staartvin** [comment]
The dev build is ready, you can download it now if you want.

**Comment by Jebula999** [comment]
It's not working out....

I think i might have to just rollback my server a few days to get the old
setup back.
But after updating STATS, it has messed up my SQL database in the process
and now all that info is gone. so this whole process has gone horribly
wrong.


I have another error for you though, http://pastebin.com/WJwKq2iw

**Comment by Staartvin** [comment]
What version were you trying? You should try dev build 396.

**Comment by Jebula999** [comment]
I am, i downloaded and added the latest DEV build you told me to try out

**Comment by Staartvin** [comment]
And it is still giving the errors? Could you provide the new log with the 396 version?

**Comment by Jebula999** [comment]
Here is the log file:
https://www.dropbox.com/s/ohduw55iqnihrf8/latest.log?dl=0

But on a side note, STATS 3 is completely broken and the owner even
recommends going back to V2.
So i am going to change to statsV2 and go back to an old version of AR till
it all works.

**Comment by Staartvin** [comment]
The error you're geting from Autorank is due to your config not properly formatted. The other errors are from Stats and they are caused because something is wrong with the database. You can reset your database and let Stats (3.0) recreate it and run Autorank 396. Or you can revert back to an older version.

**Comment by Jebula999** [comment]
Ill have to revert back to an older version, there is over a year of stats
saved in the database which i cannot lose.

**Comment by Jebula999** [comment]
I have ran through my config file i don't know how many times, and cannot
find anything wrong with the format

**Comment by Staartvin** [comment]
Could you provide your latest config?

**Comment by Jebula999** [comment]
https://www.dropbox.com/s/ir87uk45fskalhi/AdvancedConfig.yml?dl=0

**Comment by Staartvin** [comment]
Ah, you were getting that error when you were trying to set the global time of a player. '10/' is not a valid time.

**Comment by Jebula999** [comment]
That's the thing.. I didn't run any commands. I simply started the server
and let players join... And in some instances that error accurs.

But I have downgraded stats and AR and now I have bigger problems..

I'm getting really over this, I try fix one thing and another breaks. And
its just getting worse..

I will try a few things and what not and will let you know if something
comes up or I need your help again.

Thank you for the time and effort you have put in to help me.
